gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
```

Comment:
I think you need to execute `./gradlew wrapper --gradle-version 8.12.1`.

Then, the jar and the two scripts will also be updated.

Reply:
Thanks for the recommendation @hpoettker, updated!